code/controllers/configuration/entries/general.dm

```diff
@@ -600,3 +600,6 @@
 /datum/config_entry/string/elasticsearch_metrics_apikey
 
 /datum/config_entry/flag/enable_mrat
+
+/datum/config_entry/flag/cache_assets
+	config_entry_value = TRUE
```

Comment:
Let's move this into `code\controllers\configuration\entries\resources.dm` to stay consistent

Reply:
Given how unintuitive it is to enable a feature by commenting it out in the config, let's default this to off instead and treat it like the other flags.